packages/ui-dom-utils/src/getComputedStyle.ts

```diff
@@ -35,16 +36,20 @@ import { canUseDOM } from './canUseDOM'
  * specified element
  * @module getComputedStyle
  *
- * @param {ReactComponent|DomNode} el - component or DOM node
- * @returns {Object} object containing css properties and values for the element
+ * @param { Node | Window | React.ReactElement | function } el - component or DOM node
+ * @returns { Object } object containing css properties and values for the element
  */
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-function getComputedStyle(el) {
-  let style = {}
+function getComputedStyle(
+  el: Node | Window | React.ReactElement | ((...args: any[]) => any)
+) {
+  let style: CSSStyleDeclaration | Record<string, never> = {}
```

Comment:
Not sure about this `never` type here, it seems like a logical error to me.
https://www.typescriptlang.org/docs/handbook/basic-types.html#never

If we don't know the exact type `unknown` would be better I think.